src/server/services/wfs/qgswfsparameters.cpp

```diff
@@ -76,18 +76,23 @@ namespace QgsWfs
     if ( val.isEmpty() )
       return theList;
 
-    QRegExp rx( exp );
-    if ( rx.indexIn( val, 0 ) == -1 )
-    {
+    if ( exp.isEmpty() )
       theList << val;
-    }
     else
     {
-      int pos = 0;
-      while ( ( pos = rx.indexIn( val, pos ) ) != -1 )
+      QRegExp rx( exp );
```

Comment:
Ideally this would be QRegularExpression -- I suspect QRegExp will be removed in Qt6.